src/System.IO.FileSystem/src/System/IO/FileSystem.Unix.cs

```diff
@@ -56,7 +114,7 @@ public static void ReplaceFile(string sourceFullPath, string destFullPath, strin
 
                 // Now that the backup is gone, link the backup to point to the same file as destination.
                 // This way, we don't lose any data in the destination file, no copy is necessary, etc.
```

Comment:
Nit: this comment is now a little misleading, in particular the "no copy is necessary" part.